src/main/scala/strawman/collection/immutable/HashMap.scala

```diff
@@ -28,8 +30,9 @@ import java.lang.{Integer, String, System}
 @SerialVersionUID(2L)
 sealed trait HashMap[K, +V]
   extends Map[K, V]
-     with MapOps[K, V, HashMap, HashMap[K, V]]
-     with Serializable {
+    with MapOps[K, V, HashMap, HashMap[K, V]]
+    with Buildable[(K, V), HashMap[K, V]]
```

Comment:
So it's just because someone might write `MapOps[K, V, Map, Map[K, V]]` that we don't want to be `Buildable`?